aws-lc-sys/builder/main.rs

```diff
@@ -126,39 +128,42 @@ fn prefix_string() -> String {
     format!("aws_lc_{}", VERSION.to_string().replace('.', "_"))
 }
 
-#[cfg(any(
-    feature = "bindgen",
-    not(any(
-        all(
-            any(target_arch = "x86_64", target_arch = "aarch64"),
-            any(target_os = "linux", target_os = "macos"),
-            any(target_env = "gnu", target_env = "musl", target_env = "")
-        ),
-        all(target_arch = "i686", target_os = "linux", target_env = "gnu")
-    ))
-))]
+#[cfg(feature = "bindgen")]
 fn target_platform_prefix(name: &str) -> String {
     format!("{}_{}", target().replace('-', "_"), name)
 }
 
 pub(crate) struct TestCommandResult {
     #[allow(dead_code)]
-    output: Box<str>,
+    stderr: Box<str>,
+    #[allow(dead_code)]
+    stdout: Box<str>,
+    executed: bool,
     status: bool,
 }
 
+const MAX_CMD_OUTPUT_SIZE: usize = 1 << 12;
```

Comment:
The intention is to avoid collecting excessive output from a build tool (i.e., I don't want to dump megabytes of text onto the console). I figure 4k of output is sufficient to diagnose most failures for the tools that we're using in our build. So far it's been enough for my development.